content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -8,17 +8,14 @@ weight: 70
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.23" state="stable" >}}
```

Comment:
We should keep it for a few releases, so that readers can easily see that it only recently graduated and they might need to check feature gates if they're actually running an older version than v1.23